### PR TITLE
Add getBuildUser helper method

### PR DESCRIPTION
**Pull request by mpi1**
This method can be used to determine real starter of pipeline, for cases where one pipeline is started from another.

### CODE REVIEW

File: vars/getBuildUser.groovy [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Source: https://gist.github.com/jaysoncena/4b18951c3c5e68a9049ad340878761e9#gistcomment-3431750
+// Credits to pere3 @ github
+def call(currentBuild=currentBuild) {
```
Comment (take-five) [inline vars/getBuildUser.groovy:3]: What happens when it's called by a cronjob?
What happens when it's called by a remote host?

Comment (mpi1) [reply vars/getBuildUser.groovy:3]: Apparently the outcome is `null`

**Comment by mpi1** [comment]
Closing this, since we don't need it after all.

**Comment by take-five** [comment]
> Closing this, since we don't need it after all.

We still need this to fetch the cause recursively from upstream jobs

**Pull request closed without merging**